deployment/v3/external/postgres/README.md

```
@@ -1,10 +1,36 @@
 
-## Install 
+## Installation Options
+
+### Option 1: Kubernetes-based Installation (Default)
 ```sh
 ./install.sh
 ```
-* A random password will get assigned for `postgres` user if you have not specified a password. The password may be obtained from Rancher console.
+
+Note:
+  * A random password will get assigned for `postgres` user if you have not specified a password. The password may be obtained from Rancher console.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)